benchmarks/units.py

```diff
@@ -68,6 +70,22 @@ def time_quantity_init_array():
     a * u.m / u.s
 
 
+def time_quantity_init_small_list():
+    """
+    https://github.com/astropy/astropy/issues/7546 reported high overhead
+    for small list.
+    """
+    b1 * u.m / u.s
```

Comment:
Note: This is not exactly as done at https://github.com/yt-project/unyt/blob/master/benchmarks/bench.py but it is more consistent with existing benchmarks for unit initialization.